### ISSUE TITLE
car2go data

### ISSUE THREAD

**Issue by ngoldman** [CodeForPortland/housing#3]
@aaronpk are you still actively collecting car2go data? a couple people at the last @CodeForPortland meetup were curious about it.

**Comment by aaronpk** [comment]
Yes I am! I have data from June 2012 to present. I believe I'm missing a couple months worth when they changed their API but it's complete other than that! There are 135 million records now... each car is reported every minute and there are about 700 cars.
